modules/slugs/includes/class-qtranslate-slug.php

```diff
@@ -957,6 +957,7 @@ private function get_page_id_by_path( $page_path, $output = OBJECT, $post_type =
         $page_path     = str_replace( '%20', ' ', $page_path );
         $parts         = explode( '/', trim( $page_path, '/' ) );
         $parts         = array_map( 'esc_sql', $parts );
+        $parts         = array_map( array( $wpdb, 'remove_placeholder_escape' ), $parts );
         $parts         = array_map( 'sanitize_title_for_query', $parts );
```

Comment:
The 3x `array_map` may be regrouped in one function (even anonymous) for faster execution in a sngle `array_map`.

Reply:
5ca8b04995b336a292be9aa2a8b50d4f8e440dc5